src/Compute/Compute/Common/ValidateBase64EncodedString.cs

```diff
@@ -15,11 +15,12 @@ public static class ValidateBase64EncodedString
         public const string UserDataEncodeNotification = "The provided UserData parameter value was not Base64 encoded. The cmdlet has automatically changed your value by Base64 encoding it.";
         public static bool ValidateStringIsBase64Encoded(string encodedString)
         {
-            if (encodedString.Contains(" ") ||
+            if (!string.IsNullOrEmpty(encodedString) &&
```

Comment:
I ended up adding this functional change because there seems to be a bug with the UserData parameter that it is sometimes registered as a Bound Parameter but with a value of Null. This is some extra defensive coding to avoid unnecessary validation run on an empty string.

Reply:
Meaning this check will see an empty string or null as base64 encoded by default.